### DISCUSSION TITLE
Next JS and Firebase Hosting Deploy

### DISCUSSION THREAD

**Discussion by Jhernandez84** [vercel/next.js#65669] [Help]
### Summary

I've been trying to deploy an app using Next JS to firebase (I'm quite noob in programming btw), but I haven't found a step by step guide to do it properly... I use to get the "welcome message blue on firebase" but the page doesn't deploy so i'm looking for help if possible to see the proper structure that I should see on my Next JS project build and how to deploy to firebase. Please keep in mind I'm diving into web apps.. I appreciate your help.

**Comment by flymauntains** [comment]
In package.json, add "export" : "next export"
```
public directory: out
firebase login
firebase init hosting
npm run build && npm run export
firebase deploy
```
If problem is continuous, contact me. I will solved asap.
